frontend/test/ui/requirements/requirements.txt

```diff
@@ -0,0 +1,10 @@
+foxpuppet==1.0a1
```

Comment:
I will be adding an install experiment test so this was just jumping ahead a bit conforming to project goals.